compiler/rustc_session/src/options.rs

```diff
@@ -1489,6 +1489,8 @@ options! {
         "import library generation tool (ignored except when targeting windows-gnu)"),
     embed_bitcode: bool = (true, parse_bool, [TRACKED],
         "emit bitcode in rlibs (default: yes)"),
+    embed_source: bool = (false, parse_bool, [TRACKED],
+        "embed source text in DWARF debug sections (default: no)"),
```

Comment:
Somebody mentioned on Zulip but this should go in the UnstableOptions `options!` macro below. New flags exist as unstable for a while so they get some time to figure out any bugs or improvements.